qa/rpc-tests/wallet_golden_5_6_0.py

```diff
@@ -0,0 +1,225 @@
+            assert_equal(self.nodes[0].getblockcount(), 200)
+
+            # Activate nu5
+            self.nodes[0].generate(1)
+            self.sync_all()
+
+            # Set up an Orchard account on node 0
+            n0account0 = self.nodes[0].z_getnewaccount()['account']
+            assert(n0account0 == 0)
+            n0ua0_0 = self.nodes[0].z_getaddressforaccount(n0account0, ['orchard'], 0)['address']
+            assert_equal(
+                    {'pools': {}, 'minimum_confirmations': 1},
+                    self.nodes[0].z_getbalanceforaccount(n0account0))
+
+            # Fund the Orchard wallet on node 0 from Node 2 coinbase
+            n2taddr = get_coinbase_address(self.nodes[2])
+            recipients = [{"address": n0ua0_0, 'amount': Decimal('9.99985')}]
+            # TODO: since we can't have transparent change, this should only require
+            # `AllowRevealedSenders`
+            opid = self.nodes[2].z_sendmany(n2taddr, recipients, 1, None, 'AllowFullyTransparent')
+            wait_and_assert_operationid_status(self.nodes[2], opid)
+
+            self.nodes[2].generate(1)
+            self.sync_all()
+
+            assert_equal(
+                    {
+                        'pools': {'orchard': {'valueZat': Decimal('999985000')}},
+                        'minimum_confirmations': 1
+                    },
+                    self.nodes[0].z_getbalanceforaccount(n0account0))
+
+            # Send funds from Orchard on node 0 to Orchard on node 2
+            n2account0 = self.nodes[2].z_getnewaccount()['account']
+            assert(n2account0 == 0)
+            n2ua0_0 = self.nodes[2].z_getaddressforaccount(n2account0, ['orchard'], 0)['address']
+            assert_equal(
+                    {'pools': {}, 'minimum_confirmations': 1},
+                    self.nodes[2].z_getbalanceforaccount(n2account0))
+
+            recipients = [{"address":n2ua0_0, "amount": Decimal('5.0')}]
+            opid = self.nodes[0].z_sendmany(n0ua0_0, recipients, 1)
+            wait_and_assert_operationid_status(self.nodes[0], opid)
+
+            self.nodes[0].generate(1)
+            self.sync_all()
+
+            assert_equal(
+                    {
+                        'pools': {'orchard': {'valueZat': Decimal('499975000')}},
+                        'minimum_confirmations': 1
+                    },
+                    self.nodes[0].z_getbalanceforaccount(n0account0))
+
+            # Set up a second account and a couple more addresses on node 0 &
+            # send the funds back and forth to establish more note commitment
+            # tree structure.
+            n0ua0_1 = self.nodes[0].z_getaddressforaccount(n0account0, ['orchard'], 1)['address']
+            n0account1 = self.nodes[0].z_getnewaccount()['account']
+            assert(n0account1 == 1)
+            n0ua1_0 = self.nodes[0].z_getaddressforaccount(n0account1, ['orchard'], 0)['address']
+            for i in range(10):
```

Comment:
```suggestion

            # We space out transactions at 15 block increments so that some of the oldest checkpoints get dropped.
            for i in range(10):
```